examples/gallery/line/vectors.py

```diff
@@ -0,0 +1,63 @@
+"""
+Cartesian, circular, and geographic vectors
+-------------------------------------------
+
+The :meth:`pygmt.Figure.plot` method can plot Cartesian, circular, and geographic vectors.
+The ``style`` parameter controls vector attributes.
+
+"""
+import numpy as np
+import pygmt
+
+# create a plot with coast, Mercator projection (M) over the continental US
+fig = pygmt.Figure()
+fig.coast(
+    region=[-127, -64, 24, 53],
+    projection="M15c",
+    frame=True,
+    borders=1,
+    area_thresh=4000,
+    shorelines="0.25p,black",
+)
+
+
+# plot 12 Cartesian vectors with different lengths
+x = np.linspace(-116, -116, 12)  # x vector coordinates
+y = np.linspace(33.5, 42.5, 12)  # y vector coordinates
+direction = np.zeros(x.shape)  # direction of vectors
+length = np.linspace(0.5, 2.4, 12)  # length of vectors
+# Cartesian vectors (v) with red pen and fill, vector head at end, and 40 degree angle for vector head
+style = "v0.2c+e+a40+gred+h0+p1p,red"
```

Comment:
Don't commit this yet! Just wondering if we should we describe this long `style` string a bit more?

```suggestion
# Cartesian vectors (v) with red pen and fill, vector head at end (+e), and 40 degree angle for vector head
style = "v0.2c+e+a40+gred+h0+p1p,red"
```

Reply:
Looks good to me, but need to break the long comment into multiple lines.

Reply:
I added some annotations like this and split the comment into two lines

Reply:
Nice, just suggested a minor edit to the change at https://github.com/GenericMappingTools/pygmt/pull/950#discussion_r585191413.